app/domain/audits/find_content.rb

```diff
@@ -0,0 +1,23 @@
+module Audits
+  class FindContent
+    def self.call(filter = Filter.new)
+      scope = Content::Query.new
+                .page(filter.page)
+                .organisations(filter.organisations, filter.primary_org_only)
+                .document_type(filter.document_type)
+                .after(filter.after)
+                .theme(filter.theme_id)
+                .scope
+
+      return scope if filter.audit_status.blank?
+
+      if filter.audit_status.to_sym == :audited
```

Comment:
We're evaluating `filter.audit_status` twice here so we could use a case statement:
 
```rb
case filter.audit_status.to_sym
when :audited
  Policies::Audited.call(scope)
when :non_audited
  Policies::NonAudited.call(scope)
else
  scope
end
```